templates/mobile/src/amplitude.js

```diff
@@ -0,0 +1,44 @@
+import { Amplitude } from '@amplitude/react-native';
+import { getVersion, getBundleId, getApplicationName } from 'react-native-device-info';
+import ApollosConfig from '@apollosproject/config';
+
+const ampInstance = Amplitude.getInstance();
+ampInstance.init(ApollosConfig.AMPLITUDE_KEY);
+
+export const track = ({ eventName, properties = null }) => ampInstance.logEvent(eventName, properties);
+
+export const identify = () => {
+    const data = client.readQuery({
+      query: gql`
+        query currentCampusId {
```

Comment:
```suggestion
        query GetAnalyticsUser {
```

I'm imagining a debug scenario where no one is logged in yet and this query errors out and we have a hard time figuring out what's going on